tests/sys/psa_crypto_mac/tests_gen_hmac.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto
+ * @defgroup    sys_psa_crypto_hmac  PSA Generic HMAC
+ *
+ *
+ * @file        tests-gen-hmac.h
+ *
+ * @author      Daria Zatokovenko <daria.zatokovenko@mailbox.tu-dresden.de>
+ *
+ */
+
+#ifndef TESTS_GEN_HMAC_H
+#define TESTS_GEN_HMAC_H
+
+#include <stdint.h>
+#include "embUnit.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief   Generates tests for hashes/sha512.h - generic hmac
```

Comment:
```suggestion
 * @brief   Generates tests for generic hmac
```

The tests are backend-agnostic, right? Same below.

Reply:
I agree, but the comment should still say which HMAC algorithm the test exercise.